lib/gssapi/mech/gss_accept_sec_context.c

```diff
@@ -264,7 +264,9 @@ gss_accept_sec_context(OM_uint32 *minor_status,
 	    *delegated_cred_handle = GSS_C_NO_CREDENTIAL;
 	_mg_buffer_zero(output_token);
 
+        _gss_mg_log(10, "gss-asc: enter");
```

Comment:
Do we need a log level to avoid logs getting too noisy?

Reply:
I think that level 10 is supposed to be for tracing only.